api/src/Entity/Camp.php

```diff
@@ -56,9 +74,13 @@ class Camp extends BaseEntity implements BelongsToCampInterface {
      * @ORM\OrderBy({"start": "ASC"})
      */
     #[Assert\Valid]
-    #[Assert\Count(min: 1, groups: ['camp:create'])]
-    #[ApiProperty(writableLink: true, example: '[{ "description": "Hauptlager", "start": "2022-01-01", "end": "2022-01-08" }]')]
-    #[Groups(['Default'])]
+    #[Assert\Count(min: 1, groups: ['Camp:create'])]
+    #[ApiProperty(
+        readableLink: false,
+        writableLink: true,
+        example: '/periods?camp=/camps/1a2b3c4d',
```

Comment:
Hier hätte ich gerne mein altes Example zurück, sonst muss ich in Swagger sehr viel tippen um ein Camp zu erstellen